Jenkinsfile

```diff
@@ -0,0 +1,249 @@
+pipeline {
+    parameters {
+        booleanParam defaultValue: false,
+                description: 'Whether to upload the packages in playground repositories',
+                name: 'PLAYGROUND'
+    }
+    options {
+        skipDefaultCheckout()
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        timeout(time: 3, unit: 'HOURS')
+    }
+    agent {
+        node {
+            label 'base-agent-v1'
+        }
+    }
+    environment {
+        NETWORK_OPTS = '--network ci_agent'
+        ARTIFACTORY_ACCESS=credentials('artifactory-jenkins-gradle-properties-splitted')
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                checkout scm
+            }
+        }
+        stage('Stash') {
+            steps {
+                sh 'mkdir staging'
+                sh 'cp -r package conf jars staging'
+                stash includes: 'staging/**', name: 'staging'
+            }
+        }
+
+        stage('Build deb/rpm') {
+            parallel {
+                        stage('Ubuntu') {
+                            agent {
+                                node {
+                                    label 'yap-agent-ubuntu-20.04-v2'
+                                }
+                            }
+                            steps {
+                                unstash 'staging'
+                                sh 'cp -r staging /tmp'
+                                sh 'sudo yap build ubuntu /tmp/staging/package'
```

Comment:
sh 'sudo yap build ubuntu /tmp/staging/package -s'
to avoid distro repo sync and make ci faster (cause there's no compilation of native stuff)